SUI/SUI/ContentView.swift

```diff
@@ -4,15 +4,172 @@
+    @State var firstPrice: Double = Constants.zeroDouble
+    @State var offsetX = Constants.zeroInt
+    @State var isOnEngine = false
+    @State var isCall = false
+    @State var isMyChoose = false
+    @State private var isEditingSlider = false
+    var cars = Constants.carsValues
+    var typeOfSpoiler = Constants.typeOfSpoilersValues
+    
+    @State private var isSharePresented = false
+    let customActivity = CustomActivityView(title: Constants.socialString, imageName: Constants.activityImageName) {
+    }
+    
+    private var carImage: some View {
+        Image(cars[segmentIndex])
+            .resizable()
+            .frame(width: Constants.imageWidth, height: Constants.imageHeight)
+            .offset(x: CGFloat(offsetX))
+            .cornerRadius(Constants.cornerRadius)
+    }
+    
     var body: some View {
         VStack {
-            Image(systemName: "globe")
-                .imageScale(.large)
-                .foregroundColor(.accentColor)
-            Text("Hello, world!")
+            NavigationView {
+                Form {
+                    Section {
+                        VStack {
+                            ZStack {
+                                RoundedRectangle(cornerRadius: Constants.cornerRadius)
+                                    .fill(.gray)
+                                    .offset(x: CGFloat(offsetX))
+                                    .padding()
+                                carImage
+                            }.animation(.spring())
+                            
+                            Picker(selection: Binding(get: {
+                                self.segmentIndex
+                            }, set: { newValue in
+                                self.segmentIndex = newValue
+                                self.offsetX = Constants.offsetXNumber
+                                self.moveBack()
+                            }), label: Text("")) {
+                                ForEach(0 ..< cars.count) {
+                                    Text(self.cars[$0]).tag($0)
+                                }
+                            }
+                            .pickerStyle(SegmentedPickerStyle())
+                            .padding()
+                            
+                            HStack {
+                                Text(Constants.frontWindowTitle)
+                                Toggle(isOn: $isOnEngine) {}
+                            }.padding()
+                            
+                            HStack {
+                                Picker(selection: $vEngineIndex) {
+                                    ForEach(0..<typeOfSpoiler.count) {
+                                        Text(self.typeOfSpoiler[$0])
+                                    }
+                                } label: {
+                                    Text(Constants.spolerTitle)
+                                }.padding()
```

Comment:
повыноси в вычисляемые свойства
```suggestion
                                Picker(selection: $vEngineIndex) {
                                    ForEach(0..<typeOfSpoiler.count) {
                                        Text(self.typeOfSpoiler[$0])
                                    }
                                } label: {
                                    Text(Constants.spolerTitle)
                                }.padding()
```